docs/monitoring-usage.rst

```diff
@@ -154,6 +154,85 @@ before you call
     projects.metricDescriptors
 
 
+Groups
+------
+
+A group is a dynamic collection of *monitored resources* whose membership is
```

Comment:
"Whose" is the only possessive relative pronoun available in English:  Shakespeare and Milton used it that way, so its good enough for us.  See http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Usage/faq0043.html